### PR TITLE
feat: Implement fast approximation of Gelu to improve performance 

### PR DESCRIPTION
**Pull request by peri044**
# Description

 Implement fast approximation of Gelu to improve performance (eg: BERT model)

## Type of change

- New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines of this project (You can use the linters)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas and hacks
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests to verify my fix or my feature
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by narendasan** [comment]
@peri044 do you think we could do this through a lowering pass instead of having to write out the TensorRT implementation?

**Comment by narendasan** [comment]
Also is this more accurate than the plugin?

File: core/lowering/passes/BUILD [hunk 1]
```diff
@@ -17,6 +17,7 @@ cc_library(
         "module_fallback.cpp",
         "op_aliasing.cpp",
         "reduce_to.cpp",
+        "reduce_gelu.cpp",
```
Comment (narendasan) [inline core/lowering/passes/BUILD:20]: I dont see reduce_gelu.cpp?

**Comment by peri044** [comment]
> Also is this more accurate than the plugin?
This is the same as the plugin. Plugin also used approximation with the commented formula
Pushed everything, now the PR is ready.

**Review by narendasan — APPROVED** [review]
LGTM Tests pass. This is really cool! Seems like we have a lot of potential in lowering

**Pull request merged**